src/main/java/io/excaliburfrc/robot/subsystems/Drive.java

```diff
@@ -19,15 +43,105 @@
   private final CANSparkMax rightFollower =
       new CANSparkMax(DrivetrainConstants.RIGHT_FOLLOWER_ID, MotorType.kBrushless);
 
+  private final Encoder rightEncoder =
+      new Encoder(
+          DrivetrainConstants.RIGHT_ENCODER_PORT_A, DrivetrainConstants.RIGHT_ENCODER_PORT_B);
+  private final Encoder leftEncoder =
+      new Encoder(DrivetrainConstants.LEFT_ENCODER_PORT_A, DrivetrainConstants.LEFT_ENCODER_PORT_B);
```

Comment:
We're using the SparkMax's encoders, accessible through `getEncoder()`

Reply:
I already did that, I forgot to commit